pkg/vere/pier.c

```diff
@@ -1872,7 +1873,48 @@ _pier_boot_make(u3_noun who,
     bot_u.use = u3nc(u3nc(wir, cad), bot_u.use);
   }
 
+  //  prepend & append additional boot enhancements to the userspace sequence
+  //
+  {
```

Comment:
Was thinking about inlining the `$prop` type definition here for clarity, but feels kind of weird/ugly to just chuck in there. Not sure what the preferred style for including a "spec" like this is.

```hoon
+$  prop  [%prop meta tier (list ovum)]
+$  meta  term
+$  tier  ?(%fore %hind)  ::  before or after userspace
```

Reply:
just putting that in a comment here is good